README.md

```diff
@@ -336,6 +336,13 @@ The following table lists the configurable parameters for this chart and their d
 | `reportsPersistence.selector`                   | Set the selector for PVs, if desired                                | `{}`                                         |
 | `reportsPersistence.accessMode`                 | Access mode for the volume                                          | `ReadWriteOnce`                              |
 | `reportsPersistence.size`                       | Size of persistent volume to request                                | `1Gi`                                        |
+| `scriptsPersistence.enabled`                    | Enable storage persistence for NetBox reports                       | `false`                                      |
+| `scriptsPersistence.existingClaim`              | Use an existing `PersistentVolumeClaim` instead of creating one     | `""`                                         |
+| `scriptsPersistence.subPath`                    | Mount a sub-path of the volume into the container, not the root     | `""`                                         |
+| `scriptsPersistence.storageClass`               | Set the storage class of the PVC (use `-` to disable provisioning)  | `""`                                         |
+| `scriptsPersistence.selector`                   | Set the selector for PVs, if desired                                | `{}`                                         |
+| `scriptsPersistence.accessMode`                 | Access mode for the volume                                          | `ReadWriteOnce`                              |
+| `scriptsPersistence.size`                       | Size of persistent volume to request                                | `1Gi`                                        |
```

Comment:
A default of `1Gi` seems kinda massive for just scripts, isn't it? I realize it's just a request for availability and not that it's going to eat up a gig automatically, but...

Reply:
That's basically the volume size granularity you get in Kubernetes. I expect most people would use ReadWriteMany volumes for this with backing stores billed by the used byte rather than provisioned storage capacity. IIRC most volume providers just don't do sub-1GiB volume size granularity, even now.

Reply:
Gotcha. I've not really been working with the cloud providers at all in my testing, just local. More k8s weirdness. :)